src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -306,6 +306,7 @@ internal static int GetWarningLevel(ErrorCode code)
                 case ErrorCode.WRN_CallerLineNumberPreferredOverCallerMemberName:
                 case ErrorCode.WRN_CallerLineNumberPreferredOverCallerFilePath:
                 case ErrorCode.WRN_DelaySignButNoKey:
+                case ErrorCode.WRN_AttributeIgnoredWhenPublicSigning:
```

Comment:
Warning levels are used here: https://github.com/dotnet/roslyn/blob/d8f45474d9e5436a10865457817d837f80777c4f/src/Compilers/CSharp/Portable/Compilation/CSharpDiagnosticFilter.cs#L115 any error above the number specified with `/warn:_` is suppressed. If there are warnings missing, I'm pretty sure that's a bug, since it would default to 0 (which is documented as reserved for errors only, not warnings). (The parameter `diagnosticWarningLevel` in that link comes from the function modified above, through a complex maze of juggling values around)